### ISSUE TITLE
FIXME WHEN COORDINATES REFACTORED

### ISSUE THREAD

**Issue by dschwoerer** [boutproject/BOUT-dev#2727]
Should that have been fixed?

https://bout-dev.readthedocs.io/en/stable/user_docs/new_in_v5.html#fixme-when-coordinates-refactored

**Comment by ZedThree** [comment]
There's some work currently in place to refactor `Coordinates` that will hopefully eliminate the need for `geometry()` entirely, so I think that's still the case.

**Comment by dschwoerer** [comment]
But that should then be "New in 5.2.0"?

**Comment by ZedThree** [comment]
No, the text is correct for 5.0.0 (the section heading aside). The calls to `geometry()` have been removed from the `Coordinates` ctors, so it must be called manually -- but this is already the case if the user modifies any of the metrics.

**Comment by dschwoerer** [comment]
Ah, now I get your point. So we should probably change "FIXME WHEN COORDINATES REFACTORED" to "Explicit call to `[Coordinates::geometry()](https://bout-dev.readthedocs.io/en/stable/_breathe_autogen/file/coordinates_8hxx.html#_CPPv4N11Coordinates8geometryEbb) required"?
